fix: fix and update the tutorials for `0.17` (#744)
cosmos/sdk-tutorials: blog-legacy/tutorial/01-index.md

After `starport` is installed, use it to create the initial app structure inside a directory named `blog`:

```
starport scaffold chain github.com/example/blog --sdk-version launchpad
```

> One of the main features of Starport is code generation. The command above has generated a directory structure with a working blockchain application. Starport can also add data types to your app with `starport type` command. To see it in action, follow the poll application tutorial. In this guide, however, we'll create those files manually to understand how it all works under the hood.

Reviewer: ```suggestion
One of the main features of Starport is code generation. The preceding command has generated a directory structure with a working blockchain application. You can also add data types to your app with the `starport type` command. To see code generation in action, follow the poll application tutorial. In this guide, however, we'll create those files manually to learn how Starport code generation works.
```

Reviewer: - instead of "above" let's use preceding, per style standards https://developers.google.com/style/word-list#letter-a 
- instead of having Starport be the actor, let's have the user "you" add the data types
- instead of "it" let's identify the subject "code generation"
- I know it's tempting to use slang, but "under the hood" is colloquial, so can we just say "to learn..." ?

To launch your application run:

```
starport chain serve
```

> This command installs dependencies, builds and initializes the app, and runs servers. You can also do it manually:

Reviewer: ```suggestion
This command installs dependencies, builds and initializes the app, and runs servers. You can also do these tasks manually:
```
